components/script/dom/webidls/IDBFactory.webidl

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+/*
+ * The origin of this IDL file is
+ * https://w3c.github.io/IndexedDB/#idbfactory
+ *
+ */
+
+// https://w3c.github.io/IndexedDB/#idbfactory
+partial interface mixin WindowOrWorkerGlobalScope {
+  [SameObject] readonly attribute IDBFactory indexedDB;
```

Comment:
This is missing a Pref annotation, which is why the tests are changing behaviour.